extensions/wikia/ThemeDesigner/UploadFaviconFromFile.class.php

```diff
@@ -7,48 +8,48 @@ class UploadFaviconFromFile extends UploadFromFile {
 	// TODO: Are these valid error codes?
 	const FILEDIMENSIONS_ERROR = 30;
 	const FILETYPE_ERROR = 31;
-	
-	public function verifyUpload(){
+
+	public function verifyUpload() {
 		$this->getTitle(); //will fill in final destination and extension
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Move this trailing comment on the previous empty line. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS139)